CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.1.0] - 12/09/2017
+### Added
+- Initial gem without release
+
+## [0.2.0] - 04/21/2020
+### Added
+- Add jenkins pipeline
+- Initial release
+
+## [0.2.1] - 07/09/2020
+### Added
```

Comment:
This should be "Fixed" since this is a patch bump/bug fix

Reply:
Renamed it to `Changed` based on the precedent in the example changelog here
https://keepachangelog.com/en/1.0.0/

Looks like they stick to: Added, Changed, Removed